util.c

```diff
@@ -4693,26 +4693,13 @@ Perl_seed(pTHX)
      * value from (tv_sec * SEED_C1 + tv_usec).  The multipliers should
      * probably be bigger too.
      */
-#if RANDBITS > 16
```

Comment:
I'd like to see this check replaced by something like
```c
STATIC_ASSERT_DECL(RANDBITS == 48); /* hardcoded in Configure */
```
so we don't silently get garbage if someone changes it.

Reply:
I'm willing to make this change, but I really don't think it's neccessary. I sent an email to p5p looking for clarification, but I don't show that we even use `RANDBITS` anywhere in the code. Pretty sure this is an artifact of some really old code. Regardless of what `RANDBITS` is set to, the seeding and PRNG output the same value.